apps/functions-subscription/src/adapters/azure/cosmosdb/activation-job.ts

```diff
@@ -29,5 +29,25 @@ export const makeActivationJobCosmosContainer = (
         ),
         TE.mapBoth(cosmosErrorToDomainError, () => job),
       ),
+    update: (trialId, update) =>
+      pipe(
+        TE.tryCatch(
+          () =>
+            container.item(trialId, trialId).patch([
+              {
+                op: 'replace',
+                path: '/usersToActivate',
+                value: update.usersToActivate,
```

Comment:
**issue**: `usersToActivate` is defined as an optional parameter; what happens if that parameter is `undefined`?